app/components/ui-accordion.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
```

Comment:
@niranjan94 
This fix failed to work or me, nested accordion does not work. Please guide. Thanks.